### ISSUE TITLE
OmniParser Incorrectly Identifies Content of Chinese Icons.

### ISSUE THREAD

**Issue by lecgge** [microsoft/OmniParser#199]
Does the training data include app icons from mainland China?

**Comment by sw926** [comment]
Maybe you can modify the OCR recognition to support Chinese by editing the utils.py file.
``` python
reader = easyocr.Reader(['ch_sim'])
paddle_ocr = PaddleOCR(
    lang='ch',  # other lang also available
    use_angle_cls=False,
    use_gpu=False,  # using cuda will conflict with pytorch in the same process
    show_log=False,
    max_batch_size=1024,
    use_dilation=True,  # improves accuracy
    det_db_score_mode='slow',  # improves accuracy
    rec_batch_num=1024)
```

**Comment by lecgge** [comment]
> Maybe you can modify the OCR recognition to support Chinese by editing the utils.py file.
> 
> reader = easyocr.Reader(['ch_sim'])
> paddle_ocr = PaddleOCR(
>     lang='ch',  # other lang also available
>     use_angle_cls=False,
>     use_gpu=False,  # using cuda will conflict with pytorch in the same process
>     show_log=False,
>     max_batch_size=1024,
>     use_dilation=True,  # improves accuracy
>     det_db_score_mode='slow',  # improves accuracy
>     rec_batch_num=1024)

Thanks for your help, it solved my problem
